### ISSUE TITLE
[grouping rules] Grouped destination not working when using batching

### ISSUE THREAD

**Issue by frbattid** [telefonicaid/fiware-cygnus#675]
Real example:

Grouping rules:
```
{
    "grouping_rules": [
        {
            "id": 1,
            "fields": [
                "entityId"
            ],
            "regex": "DEVMATCH1",
            "destination": "newdest1",
            "fiware_service_path": "newservpath1"
            "sinks": [
                "mysql-sink"
            ]

        },
        {
            "id": 2,
            "fields": [
                "entityType"
            ],
            "regex": "TYPEMATCH2",
            "destination": "newdest2",
            "fiware_service_path": "newservpath2"
        }
    ]
}
```

Notification:
```
{
  "subscriptionId": "566febbf87137579fa936e30",
  "originator": "localhost",
  "contextResponses": [
    {
      "contextElement": {
        "type": "thing",
        "isPattern": "false",
        "id": "DEVMATCH1",
        "attributes": [
          {
            "name": "temperature",
            "type": "centigrade",
            "value": "1340",
            "metadatas": [
              {
                "name": "MetaTest",
                "type": "TestType1",
                "value": "UPDATED"
              }
            ]
          }
        ]
      },
      "statusCode": {
        "code": "200",
        "reasonPhrase": "OK"
      }
    }
  ]
}
```

Log:

```
2015-12-15 05:30:50,257 (SinkRunner-PollingRunner-DefaultSinkProcessor) [INFO - com.telefonica.iot.cygnus.sinks.OrionMySQLSink.persistAggregation(OrionMySQLSink.java:430)] [mysql-sink] Persisting data at OrionMySQLSink. Database (smartvalencia), Table (newservpath1_devmatch1_thing), Fields ((recvTime,fiwareservicepath,entityId,entityType,temperature,temperature_md)), Values (('2015-12-15T10:30:24.659Z','newservpath1','DEVMATCH1','thing','1340','[{"name":"MetaTest","type":"TestType1","value":"UPDATED"
```

**Comment by rbl219** [comment]
Has been tested OK Cygnus rpm generated with branch  bug/675_grouped_destination_not_working_with_batches


Pending merge with branch release/0.11.0

**Comment by frbattid** [comment]
Fixed in PR https://github.com/telefonicaid/fiware-cygnus/pull/677

**Comment by frbattid** [comment]
Exceptionally, this has also been fixed for release/0.11.0 in PR https://github.com/telefonicaid/fiware-cygnus/pull/701
